_layouts/team-page.html

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+---
+
+<div class="team-profile">
```

Comment:
On mobile there's not much space between the image and the nav - not sure if this is a larger problem though.

Reply:
Yeah I was trying to figure out how to get this spacing right. I didn't want to make a whole other stylesheet for the teams profiles in this PR since things are already getting a little chunky. Will probably push it back to the next PR.

Reply:
You can prob just add `margin-top: 50px` or something when we're on mobile for now in `.team-profile`? But yeah, it may involve us looking at the container in the future.